protinfo.go

```diff
@@ -18,6 +18,10 @@ type Protinfo struct {
 
 // String returns a list of enabled flags
 func (prot *Protinfo) String() string {
+	if prot == nil {
+		return ""
```

Comment:
I guess it will be better to return something like "<nil>" in order to distinguish from an object that is actually properly pointed but has no flag enabled

Reply:
In stdlib in such places we have more often used "<nil>".

Reply:
Thanks for reviewing! Branch updated to make the function return `"<nil>"`.